### PR TITLE
archlinux: Update main content, issue tracker link and maintainer list

### LINKED ISSUE #1785: archlinux has moved its docker repository
https://github.com/archlinux/archlinux-docker is now a read-only mirror of https://gitlab.archlinux.org/archlinux/archlinux-docker, so the "Where to file issues" URL in the auto-generated README is incorrect.

### PR DESCRIPTION
**Pull request by hashworks**
WIP: This can be merged as soon as Arch Linux switches to weekly automated PRs to `official-images`, which will happen in the next few days.
EDIT: [Done](https://github.com/docker-library/official-images/pull/9009)

This also resolves #1785. However, our GitLab is not open for registrations yet™. This will change soon.

### CODE REVIEW

File: archlinux/content.md [hunk 1]
```diff
@@ -2,31 +2,33 @@
...
 
 ## Updating
 
-Arch Linux is a rolling release distribution, so a full update is recommended when installing new packages. In other words, we suggest to either execute `RUN pacman -Syu` immediately after your `FROM` statement or as soon as you docker run into a container.
+Arch Linux is a rolling release distribution, so a full update is recommended when installing new packages. In other words, we suggest to either execute `RUN pacman -Syu` immediately after your `FROM` statement or as soon as you `docker run` into a container.
 
 ## How It's Made
 
-You can build this image with the tools on the [Github repository](%%GITHUB-REPO%%) using the included makefile.
+You can build this image with the tools on the [GitLab repository](https://gitlab.archlinux.org/archlinux/archlinux-docker) using the included makefile.
```
Comment (klausenbusk) [inline archlinux/content.md:34]: Maybe we should say `ArchLinux's GitLab repository` here? Just writing GitLab kinda implies gitlab.com

Comment (hashworks) [reply archlinux/content.md:34]: fair enough

File: archlinux/issues.md [hunk 2]
```diff
@@ -0,0 +1 @@
+https://gitlab.archlinux.org/archlinux/archlinux-docker/issues
```
Comment (tianon) [inline archlinux/issues.md:1]: Should we wait to merge this until this is something users can actually contribute to, or is this containing at least some information a net win over the current unusable link? :sweat_smile:

Comment (hashworks) [reply archlinux/issues.md:1]: Actually, yes! The current issue tracker is closed AND contains no issues. The GitLab link is just closed.

**Pull request merged**